### PR TITLE
TMS320C64x instruction names

### PR DESCRIPTION
**Pull request by catenacyber**
Found by oss-fuzz
https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=12953

### CODE REVIEW

**Comment by catenacyber** [comment]
And another one bug found by oss-fuzz, undefined shift
https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=12954

**Comment by aquynh** [comment]
@fotisl, please ack

**Comment by fotisl** [comment]
The first two patches seem to be ok.

File: arch/TMS320C64x/TMS320C64xMapping.c [hunk 1]
```diff
@@ -1718,17 +1718,156 @@ void TMS320C64x_get_insn_id(cs_struct *h, cs_insn *insn, unsigned int id)
 }
 
 #ifndef CAPSTONE_DIET
+//grep TMS320C64X_INS include/capstone/tms320c64x.h | awk '{print "{"$1 "\""tolower(substr($1, 16, length($1)-16))"\"""},"}'
 static name_map insn_name_maps[] = {
-	{ TMS320C64X_INS_INVALID, NULL },
-
-	{ TMS320C64X_INS_ADD, "add" },
-	{ TMS320C64X_INS_NOP, "nop" },
+	{TMS320C64X_INS_INVALID,NULL},
```
Comment (aquynh) [inline arch/TMS320C64x/TMS320C64xMapping.c]: could you put a space after the comma here, and also below?

Comment (catenacyber) [reply arch/TMS320C64x/TMS320C64xMapping.c]: Like this ?

File: arch/TMS320C64x/TMS320C64xMapping.c [hunk 2]
```diff
@@ -1720,152 +1720,152 @@ void TMS320C64x_get_insn_id(cs_struct *h, cs_insn *insn, unsigned int id)
...
+	{TMS320C64X_INS_XOR, "xor"},
+	{TMS320C64X_INS_XPND2, "xpnd2"},
+	{TMS320C64X_INS_XPND4, "xpnd4"},
+	{TMS320C64X_INS_IDLE, "idle"},
+	{TMS320C64X_INS_MV, "mv"},
+	{TMS320C64X_INS_NEG, "neg"},
+	{TMS320C64X_INS_NOT, "not"},
+	{TMS320C64X_INS_SWAP2, "swap2"},
+	{TMS320C64X_INS_ZERO, "zero"},
+	{TMS320C64X_INS_ENDING, "ending"},
```
Comment (aquynh) [inline arch/TMS320C64x/TMS320C64xMapping.c]: this ENDING is just to mark end of list of instructions, but not a real instruction, so it should not be here.
please remove it

File: arch/TMS320C64x/TMS320C64xMapping.c (hunk 1 above)
Comment (aquynh) [reply arch/TMS320C64x/TMS320C64xMapping.c]: yes, this is just about coding style

File: arch/TMS320C64x/TMS320C64xMapping.c (hunk 2 above)
Comment (catenacyber) [reply arch/TMS320C64x/TMS320C64xMapping.c]: indeed

**Comment by aquynh** [comment]
merged, thanks!

**Pull request merged**